### ISSUE TITLE
Using services in a bundle configuration

### ISSUE THREAD

**Issue by emr** [symfony/symfony#27078]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | no
| BC Break report? | no
| Symfony version  | 3.4

—

```yml
easy_admin:
    entities:
        MyEntity:
            class: "@my_config_service.getMyEntityClass()"
            ...
```

Does not compiled expressions in bundle configuration.

I tried to make php configuration:
```php
$container->loadFromExtension('easy_admin', [
    'entities' => [
        'MyEntity' => [
            'class' => $container->get('my_config_service')->getMyEntityClass()
        ]
    ]
]);
```
I take this error: **`You have requested a non-existent service "my_config_service". `**

—

How to inject my service in a bundle configuration?
Is there an alternative way?

(btw, sorry for my bad english)

**Comment by stof** [comment]
supporting expressions in the semantic config of a bundle is something that each bundle has to support on its own. Expressions in the container are represented using a `Expression` value object. The YAML parser (or the XML parser for that matter) does not have a built-in way to create such objects. So it requires each place processing the config to convert the received config to an Expression object when it wants to support it.

Btw, if the bundle expects to be able to use this setting during the compile-time of the container, using expressions accessing services will not be an option, as you cannot get services (at least not safely) at that time.

**Comment by emr** [comment]
@stof yes, I know that. But I need a way to inject service into a bundle configuration. Or modify another bundle configuration in my bundle.

**Comment by emr** [comment]
@stof Thank you. This was useful for me.
But there is a new problem; I want to execute `prepend()` after another extension was loaded.
So how do I set to the priority?
I tried to do it like this:
```php
public function prepend(ContainerBuilder $container)
{
    $container->getExtension('framework')->load(
        $container->getExtensionConfig('framework'),
        $container
    );
    ...
```
it worked. Unsurprisingly, I take this: **`"Symfony\Component\Console\Command\Command" has already been autoconfigured and merge() does not support merging autoconfiguration for the same class/interface. `**
